src/lib/intl.js

```diff
@@ -0,0 +1,37 @@
+import { IntlProvider, addLocaleData } from 'react-intl';
+import intlLocaleData from 'react-intl-locale-data'; // eslint-disable-line
```

Comment:
Just FYI, this doesn't exist in react-intl@3.x.

Reply:
Correct, I have created the stories to upgrade to react-intl@3.x for Preview and Annotations.